bugsnag-android-core/src/main/java/com/bugsnag/android/Session.java

```diff
@@ -61,7 +60,9 @@ static Session copySession(Session session) {
     Session(File file, Notifier notifier, Logger logger) {
         this.file = file;
         this.logger = logger;
-        this.notifier = notifier;
+        Notifier copy = new Notifier(notifier.getName(), notifier.getVersion(), notifier.getUrl());
+        copy.setDependencies(new ArrayList<>(copy.getDependencies()));
```

Comment:
Copy-paste-o:

```suggestion
        copy.setDependencies(new ArrayList<>(notifier.getDependencies()));
```

This shows the unit test didn't cover as much as it could - it should check the behaviour with dependent notifiers set.

Reply:
Updated to test this with the dependent notifier set.